app/assets/stylesheets/components.scss

```diff
@@ -7,7 +7,7 @@
 
 .dgu-highlight {
   background: $grey-3;
-  padding: em(2) em(5) em(2) em(5);
+  padding: 2px govuk-spacing(1);
```

Comment:
What's the need for the 2px? Is it needed anywhere else? Ideally we don't have bare numbers in the main css, and might assign it to a var, but maybe not worth it for one outlier.